src/SFA.DAS.EmployerFeedback.Data/Configuration/EmployerFeedbackResultConfiguration.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using EmployerFeedbackResultEntity = SFA.DAS.EmployerFeedback.Domain.Entities.EmployerFeedbackResult;
+using System.Diagnostics.CodeAnalysis;
+
+namespace SFA.DAS.EmployerFeedback.Data.Configuration
+{
+    [ExcludeFromCodeCoverage]
+    public class EmployerFeedbackResultConfiguration : IEntityTypeConfiguration<EmployerFeedbackResultEntity>
+    {
+        public void Configure(EntityTypeBuilder<EmployerFeedbackResultEntity> entity)
+        {
+            entity.ToTable("EmployerFeedbackResult");
+            entity.HasKey(e => e.Id).IsClustered(false);
+            entity.Property(e => e.Id).HasDefaultValueSql("NEWSEQUENTIALID()");
```

Comment:
I am not sure if this is required possibly not and the adding to the context will get you an Id you can use and then later save